### ISSUE TITLE
Build issues on android

### ISSUE THREAD

**Issue by YuliaGrigorieva** [invertase/jet#12]
Hello!

I have faced with the following error while building for android:

```
A problem occurred configuring project ':jet'.
> Could not resolve all files for configuration ':jet:classpath'.
   > Could not find manifest-merger.jar (com.android.tools.build:manifest-merger:26.0.1).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/build/manifest-merger/26.0.1/manifest-merger-26.0.1.jar
   > Could not find ddmlib.jar (com.android.tools.ddms:ddmlib:26.0.1).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/ddms/ddmlib/26.0.1/ddmlib-26.0.1.jar
   > Could not find dvlib.jar (com.android.tools:dvlib:26.0.1).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/dvlib/26.0.1/dvlib-26.0.1.jar
   > Could not find common.jar (com.android.tools:common:26.0.1).
     Searched in the following locations:
         https://jcenter.bintray.com/com/android/tools/common/26.0.1/common-26.0.1.jar
```

Not sure what caused this issue, however the workaround recommended [here](https://stackoverflow.com/questions/50563783/could-not-find-manifest-merger-jar-com-android-tools-buildmanifest-merger26-1) helped me.

Is it possible to invert order of repositories in the next release? 

Jet versions: v0.2.0, v0.2.1
React-native: 0.57.1
Gradle: 4.4 

Thanks,
Yulia Grigorieva

**Comment by Salakar** [comment]
Hey, thanks for the report and suggested fix, I'll add this for the next patch release this week. 👌

**Comment by Salakar** [comment]
Hey, this is fixed in 0.3.0. Thanks
